wp-content/themes/theme-publicsource/css/style.css

```diff
@@ -666,18 +666,31 @@ body.normal.page.page-template-page-clean article.page > header h1 {
   .nav-shelf li.donate {
     display: block;
   }
-  #header-extras li.donate {
-    display: none;
-  }
 }
 @media (min-width: 391px) {
-  .nav-shelf li.donate {
-    display: none;
+  #header-extras li.donate {
+    display: block;
   }
```

Comment:
Are we okay with it appearing twice when the menu is opened?

![Screen Shot 2020-05-04 at 16 18 47](https://user-images.githubusercontent.com/1754187/81009623-fba2bc80-8e22-11ea-9fb5-525d67680330.png)

Reply:
From @MirandaEcho in slack:

> yes to being in both, that's fine